x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -201,18 +199,12 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		}
 
 		// pairing is valid, we can pay provider for work
-		coinsPerCu := k.Keeper.MintCoinsPerCU(ctx)
```

Comment:
remove mintcoinspercu from protobuff, make it reserved

Reply:
done. see https://github.com/lavanet/lava/commit/eca2852a8a3ff3cf7e4f1c9c3351591ed83ddc03